apps/aehttp/src/ws_handler.erl

```diff
@@ -11,13 +11,21 @@
 
 -define(GPROC_KEY, {p, l, {?MODULE, broadcast}}).
 
+-define(SUBSCRIBE_EVENTS, [block_created %% node mined a block
+                          ]).
+
 init({tcp, http}, _Req, _Opts) ->
     {upgrade, protocol, cowboy_websocket}.
 
 websocket_init(_TransportName, Req, _Opts) ->
     case jobs:ask(ws_handlers_queue) of
         {ok, JobId} ->
             gproc:reg(?GPROC_KEY),
+            lists:foreach(
```

Comment:
Note that you could have done this for brevity
`lists:foreach(fun aec_events:subscribe/1, ?SUBSCRIBE_EVENTS)`